packages/atomic/src/components/atomic-pager-v1/atomic-pager.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  Pager,
+  PagerState,
+  buildPager,
+  buildSearchStatus,
+  SearchStatus,
+  SearchStatusState,
+} from '@coveo/headless';
+import {
+  Bindings,
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+import ArrowRight from '../../images/arrow-right.svg';
+import {Button} from '../common/button';
+
+/**
+ * The `atomic-pager` provides buttons that allow the end user to navigate through the different result pages.
+ *
+ * @part buttons - The list of buttons.
+ * @part previous-button - The previous button.
+ * @part next-button - The next button.
+ * @part page-button - The page button.
+ * @part active-page-button - The active page button.
+ */
+@Component({
+  tag: 'atomic-pager-v1', // TODO: remove v1
+  styleUrl: 'atomic-pager.pcss',
+  shadow: true,
+})
+export class AtomicPager implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  private pager!: Pager;
+  public searchStatus!: SearchStatus;
+
+  @BindStateToController('pager')
+  @State()
+  private pagerState!: PagerState;
+  @BindStateToController('searchStatus')
+  @State()
+  private searchStatusState!: SearchStatusState;
+  @State() error!: Error;
+
+  @Event({
+    eventName: 'atomic/scrollToTop',
+  })
+  private scrollToTopEvent!: EventEmitter;
+
+  /**
+   * Specifies how many page buttons to display in the pager.
+   */
+  @Prop() numberOfPages = 5;
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+    this.pager = buildPager(this.bindings.engine, {
+      options: {numberOfPages: this.numberOfPages},
+    });
+  }
+
+  private scrollToTop() {
+    this.scrollToTopEvent.emit();
+  }
+
+  private get previousButton() {
+    return (
+      <Button
+        style="outline-primary"
+        ariaLabel={this.bindings.i18n.t('previous')}
+        onClick={() => {
+          this.pager.previousPage();
+        }}
+        part="previous-button"
+        disabled={!this.pagerState.hasPreviousPage}
+      >
+        <atomic-icon
+          icon={ArrowRight}
+          class="w-5 transform rotate-180"
+        ></atomic-icon>
+      </Button>
+    );
+  }
+
+  private get nextButton() {
+    return (
+      <Button
+        style="outline-primary"
+        ariaLabel={this.bindings.i18n.t('next')}
+        onClick={() => {
+          this.pager.nextPage();
+        }}
+        part="next-button"
+        disabled={!this.pagerState.hasNextPage}
+      >
+        <atomic-icon icon={ArrowRight} class="w-5"></atomic-icon>
+      </Button>
+    );
+  }
+
+  private get pages() {
+    const pages = this.pager.state.currentPages;
+    return pages.map((page) => this.buildPage(page));
+  }
+
+  private buildPage(page: number) {
+    const isSelected = this.pager.isCurrentPage(page);
+    return (
+      <li>
+        <Button
+          ariaCurrent={isSelected ? 'page' : undefined}
```

Comment:
Is it okay to have `undefined` here ? 

Is that valid HTML/valid accessibility property ?

Im seeing `aria-current="false"` on the web: https://www.aditus.io/aria/aria-current/

Reply:
It won't be added to the attributes if it's undefined with stencil's jsx https://github.com/coveo/ui-kit/blob/master/packages/atomic/src/components/common/button.tsx#L73 at least that how it appears to behave 😄  
but false could be an option too here perhaps

Reply:
Instead of `aria-current`, it may be preferable to use `aria-pressed` and set it to `"true"` or `"false"` (only the previous and next button should be `undefined`, since `aria-pressed` makes an element a toggle button).

Alternatively, this could be made into a radiogroup, which would probably be the most accessible option but takes a lot more work.

If you stick to `aria-current`, I don't think we should set it to `false`.